index.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
           enabled: this._shouldIncludeFiles(),
           import: ['index.js']
         };
+      },
+      jquery: {
+        import: ['dist/jquery.js'],
```

Comment:
What happens here when your app has not opted out of using jQuery? Do we then bundle jQuery twice?

Reply:
I've just used this branch on a project with `jquery@1.11.3` included. As a result both `jquery` versions are bundled into `vendor.js`.

The interesting thing is that `1.11.3` is still used by `ember` and `v3.2.1` by `e-c-p-o` so their are no in conflict.

Reply:
all the jquery plugins should be included before `e-c-p-o`. Otherwise they will extend `e-c-p-o`'s jquery instead of `Ember.$` :/